### PR TITLE
Create r-geos image

### LINKED ISSUE #8: Create r-geos image
Create an `r-geos` image so cumbersome installation of geo/gis related packages does not take so long.

### PR DESCRIPTION
**Pull request by chamaoskurumi**
closes #8 

- We dont need an [`aws.config`]( `https://github.com/INWTlab/r-docker/blob/latest/r-batch/aws.config) file like in the case for `r-batch` right?
- Do I need to create an example in the examples folder with a `main.R` script that includes some kind of geo operation, maybe using a function from the `sf::` package?
- I changed the ReadMe beyond `r-geos` because it looked somewhat outdated.
- [Jenkins fails ](https://inwt-vmeh2.inwt.de/jenkins/blue/organizations/jenkins/r-docker/detail/feature%2F8_rgeos_image/1/pipeline) :man_facepalming: - because of 

image

Does that mean my feature branch has a name that cannot be handeld well by docker? Should I rename it to `feature_8_rgeos_image` so there is no `/` in the name?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,17 +27,22 @@ The container from this project can be found at:
 -   Packages for data manipulation (data.table, dplyr, tidyr)
 -   Home-brewed and open source (dbtools, mctools)
 
+### r-geos
+
+-   Starts from r-batch
+-   Installs Linux libraries necessary for running geo/gis related operations.
+-   Adds geo/gis related packages (sf, stars, terra, etc.)
```
Comment (Cattes) [inline README.md]:
```suggestion
-   Adds geo/gis related r packages (sf, stars, terra, etc.)
```

File: README.md [hunk 2]
```diff
@@ -33,17 +27,22 @@ The container from this project can be found at:
...
 -   Starts from r-batch
 -   Adds shiny related packages (shiny, shinyjs, etc.)
 
-Start them using:
+### r-ver-ubuntu (deprecated)
 
-```
-docker pull inwt/r-base:3.4.4
-docker run -it inwt/r-base:3.4.4
-```
```
Comment (Cattes) [inline README.md]: Wieso hast du den "Start them using:" Block gelöscht?

**Review by Cattes — APPROVED** [review]
Sieht gut aus, ich habe nur eine Frage und eine Suggestion.

**Comment by Cattes** [comment]
> * We dont need an `aws.config` file like in the case for `r-batch` right?

I am not sure about that, but if the deployment of the Image works then we do not.

> * Do I need to create an example in the examples folder with a `main.R` script that includes some kind of geo operation, maybe using a function from the `sf::` package?

If you got time an example would be great, if not we can add one later / next time someone bases a project on this container

> Jenkins fails [...] 
> Does that mean my feature branch has a name that cannot be handeld well by docker? Should I rename it to `feature_8_rgeos_image` so there is no `/` in the name?

Yes, you should rename the branch. the docker error code 125 does not give any details about the source of the error, but your guess about the `/` in the name seems plausible

File: README.md (hunk 2 above)
Comment (chamaoskurumi) [reply README.md]: Das bezog sich auf `r-ver-ubuntu` und das ist ja deprecated. Alle images werden gleich verwendet, ich finde das verwirrt hier eher als dass es hilft.

Comment (chamaoskurumi) [reply README.md]: Zur nutzung gibt es ja die examples

Comment (chamaoskurumi) [reply README.md]: Hab es nochmal mit Michelle besprochen ich nehme es wieder anders kommertiert rein

**Pull request closed without merging**